agent/agent_test.go

```diff
@@ -277,11 +277,10 @@ func (s *AgentTestSuite) TestStatusAfterConnFail(t *C) {
 func (s *AgentTestSuite) TestStartStopService(t *C) {
 	// To start a service, first we make a config for the service:
 	qanConfig := &pc.QAN{
-		Interval:          60,         // seconds
-		MaxSlowLogSize:    1073741824, // 1 GiB
-		RemoveOldSlowLogs: true,
```

Comment:
It seems that option `RemoveOldSlowLogs` got removed... so now agent doesn't remove old slow logs?